### DISCUSSION TITLE
Welcome to awesome-roadmaps Discussions!

### DISCUSSION THREAD

**Discussion by liuchong** [liuchong/awesome-roadmaps#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by azarudeena** [comment]
https://github.com/vazeri/Embedded-Engineering-RoadMap-2018

Closer to Embedded.

Something for Hardware electronics engineering would be interesting. Like for audrino/Raspberry pi etc.

**Reply by liuchong** [reply]
Thanks, let's check that 👀

**Comment by CodingwithFawad** [comment]
Thanks
